content/200-concepts/100-components/02-prisma-client/260-client-extensions/140-shared-extensions.mdx

```diff
@@ -62,23 +63,23 @@ export default Prisma.createExtension({
 })
 ```
 
-You could then share the above code on `npm` or as a code snippet.
+You could then share the above code on `npm` or as a code snippet. When you choose a package name, we recommend that you use the same name as the extension, to make it easier to find and install.
 
 ## Install a packaged extension
 
 In your project, you can install any Prisma Client extension that another user has published to `npm`. To do so, add the following line to your application code:
 
 ```ts
-npm install prisma-extension-<name>
+npm install prisma-extension-<package-name>
 ```
 
-For example, if an available extension is called `find-or-create`, you could install it as follows:
+For example, if the package name for an available extension is `find-or-create`, you could install it as follows:
```

Comment:
I think the npm package name is `prisma-extension-find-or-create` here.

Reply:
Yep